docs/interface/content-overview.md

```diff
@@ -36,4 +36,15 @@ The "Most Recent Build" tab is simply an already expanded version of the last bu
 
 Under the "Settings" tab, you can adapt the settings of a particular repository's branch.
 
+### Builds
+
 On every build attempt, all exercises in your chapters can be tested for correctness: it checks whether running the exercise's solution happens without errors and whether the Submission Correctness Test passes, presenting the success message to the student. When you push changes to a branch, by default only the chapters that are updated are tested. This setting can be updated to test all chapters whenever a change is made, or to test no chapters whatsoever. It is suggested to not test chapters when your course is still in early development, and you don't expect the exercises to all work. Later, when your course becomes ready for production, you can switch back to testing the chapters that are updated. Note: Your changes will be updated in the corresponding DataCamp course, even though some exercises might fail.
+
+### Collaborators
+
+If you have administrative permissions, you can add three types of collaborators to a course:
+
+- An *instructor* (who is shown on the course's public page when it launches).
+- Any number of *collaborators* (who are shown next to the instructor on the course's page).
+- Any number of *silent collaborators* (who have access but aren't displayed publicly).
```

Comment:
Silent collaborators are typically the boss of the instructor, who wants to see what is happening, but doesn't do any work on the course.

Reply:
I like that the current descriptions are strictly the behaviors that result from each setting. 

(If needed, could def see "who should I add there?" being addressed on an instructor specific page)

Reply:
I don't think instructors should be adding people to this at all, should they?